documentation/onboard-form-template.md

```diff
@@ -0,0 +1,47 @@
+# Teacher services infra - Service onboarding form
+- Service name?
+   - e.g. Calculate teacher pay
+- Domains  ? e.g.
+    - school-placements-production.teacherservices.cloud
+    - getintoteaching.education.gov.uk and
+    - get-information-schools.service.gov.uk
+- Service short name? e.g. ctp
+- Repository? e.g. https://github.com/DFE-Digital/itt-mentor-services
+- Environments? e.g. develoment, staging, production
+- Review apps? Yes or No
+- Cost centre(using format cost centre/act code )? e.g. 10167/101304
+- Activity code? e.g
+    - placements: 101423
+    - track: 101056
+- List of developers who will need access to Azure? Do they use a DfE laptop or BYOD? e.g
+    - James Beagle - BYOD
+    - Clinton Morris - DfE laptop
+- GDS assessed? Yes or No
+
+- Slack channels? e.g
+    - Product: #itt-mentor-services
+    - Tech: #itt-mentor-services-devs
+- Timeline? e.g
+    - QA/Staging ASAP
+    - Prod track-and-pay April
+    - Prod school-placements TBC but likely April
+    - Timeline of the different phases: discovery, alpha, private beta, public beta, live e.g.
+        - discovery:Mid January
+        - alpha:Mid March
+        - private beta:Mid May
+        - public beta:Mid July
+        - live: End November
+        - according to https://www.gov.uk/service-manual/agile-delivery/how-the-discovery-phase-works
+
+- Github repos / Docker registry type: Open (public) by default, or indicate any requirement to make them private
+- Namespaces e.g
```

Comment:
There is no need to expand. Also it's confusing as a namespace can have multiple environments

I suggest:
- Namespaces e.g bat-qa (test cluster), bat-staging (test cluster), bat-production (production cluster)